### DISCUSSION TITLE
.NET Implementation

### DISCUSSION THREAD

**Discussion by riatdyl** [elsa-workflows/elsa-core#1099] [Q&A]
Hello,

I've already asked for this, but I need clarity with Elsa 2.

How can I implement the workflow engine to a .NET ERP. (Not .NETCore)

**Comment by sfmskywalker** [comment]
Hi Dylan,

Given that Elsa is relies on packages not supported by .NET Framework <= 4.8, you won't be able to run workflows in-process. But what you might do is run Elsa as a separate service and interact with it via HTTP, AMQP or even gRPC. HTTP and AMQP is already there for the most part, gRPC has not yet started.

When designing workflows, you might use the **Send Command** activity to send a command to your ERP, and the ERP might use the **/conductor/events/{eventName}/dispatch** or **/conductor/events/{eventName}/execute** API endpoints to send events to the workflow, which will trigger **Event Received** activities to start & resume workflows.
